src/stories/genreSelect.stories.ts

```diff
@@ -0,0 +1,40 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import GenreSelect from '../components/genreSelect/GenreSelect';
+import { fireEvent, userEvent, within } from '@storybook/testing-library';
+
+const meta: Meta<typeof GenreSelect> = {
+  title: 'GenreSelect',
+  component: GenreSelect,
+};
+export default meta;
+type Story = StoryObj<typeof meta>;
+const genreList: Genre[] = ['Horror', 'Romantic', 'Thrilling'];
+
+export const EmptyGenreList: Story = {
+  args: {
+    genreList: [],
+    selectedGenre: undefined,
+    onSelect: (genre) => console.log('Genre Selected ' + genre),
+  },
+};
+
+export const NoDefaultGenreSelected: Story = {
+  args: {
+    genreList: genreList,
+    selectedGenre: undefined,
+    onSelect: (genre) => console.log('Genre Selected ' + genre),
+  },
+};
+
+export const ChangeGenre: Story = {
+  args: {
+    genreList: genreList,
+    selectedGenre: 'Romantic',
+    onSelect: (genre) => alert('Genre Selected ' + genre),
```

Comment:
Doesnt works

Reply:
Could you please explain more ? I've having a delay of 200 milliseconds and it selects the horror genre instead of romantic and returns an alert. what should be changed ?

Reply:
![Screenshot 2024-03-10 211109](https://github.com/weam1995/ReactMentoringProject/assets/79011727/901873f4-c55b-40a7-a239-d94c1606d8fa)

Reply:
Uploading Recording 2024-03-10 211749.mp4…